build/version.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+final class FlowVersion
+{
+    public static function getVersion(): string
+    {
+        return '1.0.x-@git_commit_short@';
```

Comment:
Could you elaborate on this one? I'm not sure if I understand why it's needed

Reply:
It allows us to set the version of build phar (and binary in the end), so we could build them based on the `1.x` branch as well on other versioning, it was just an idea to match the current branch name and add git commit into the name of the version, I'm open to change this.